compiler/src/dotty/tools/backend/jvm/DottyBackendInterface.scala

```diff
@@ -727,9 +727,9 @@ class DottyBackendInterface(outputDirectory: AbstractFile, val superCallsMap: Ma
     // navigation
     def owner: Symbol = toDenot(sym).owner
     def rawowner: Symbol = {
-      originalOwner
+      originalLexicallyEnclosingClass
     }
-    def originalOwner: Symbol =
+    def originalLexicallyEnclosingClass: Symbol =
```

Comment:
I think this name is also misleading since this calls `owner` (well `maybeOwner` but that's the same thing) and then `lexicallyEnclosingClass`. I think we need to replace existing uses of `.originalOwner` by `.originalOwner.originalLexicallyEnclosingClass` and split this method appropriately into these two methods)